conda/exports.py

```diff
@@ -94,3 +94,4 @@
 get_local_urls = lambda: list(get_conda_build_local_url()) or []
 load_condarc = lambda fn: conda.base.context.reset_context([fn])
 PaddingError = conda.exceptions.PaddingError
+from conda.common.compat import CrossPlatformStLink
```

Comment:
Probably need a `# NOQA` on that line.  And then add a line below

    CrossPlatformStLink = CrossPlatformStLink

to help the IDEs out.